test/integration/apis/issue.test.ts

```diff
@@ -78,21 +78,6 @@ describe("issue", () => {
         });
     });
 
-    describe("load requests", () => {
```

Comment:
This tests should still work. I guess it is breaking because of the types update.

Reply:
This test was duplicated for some reason